convert.go

```diff
@@ -110,8 +112,38 @@ func Move(oldPath string, newPath string, out io.Writer) error {
 		if err != nil {
 			return err
 		}
-		if inf.IsDir() || fn == SHARDING_FN || fn == README_FN {
-			// we are an empty directory or generated file so just remove it
+		if inf.IsDir() {
+			indir, err := os.Open(oldPath)
+			if err != nil {
+				return err
+			}
+			defer indir.Close()
+
+			names, err := indir.Readdirnames(-1)
+			if err != nil {
+				return err
+			}
+
+			for _, n := range names {
+				p := filepath.Join(oldPath, n)
+				// part of unfinished write transaction
+				// remove it
```

Comment:
My original suggestion was to delete it. I still think that is the right thing to do. The write that created this file failed, giving the user no expectation of the data existing anywhere on disk.